.conf/Dockerfile.full

```diff
@@ -0,0 +1,49 @@
+###############################################################
+# Copyright (c) 2022 Contributors to the Eclipse Foundation
```

Comment:
for new files please specify current year

Reply:
Done